sysbuild/CMakeLists.txt

```diff
@@ -848,6 +848,62 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_post_cmake)
       set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${BINARY_DIR}/zephyr/.config)
     endif()
   endif()
+
+  # TEMPHACK: UICR generator for IronSide SE for downstream
+  set(ironside_uicr_images "")
+  set(ironside_uicr_main_image)
+  foreach(image ${POST_CMAKE_IMAGES})
+    sysbuild_get(image_board_target IMAGE ${image} VAR CONFIG_BOARD_TARGET KCONFIG)
+    if(image_board_target MATCHES "/nrf54h20/(cpuapp|cpurad)")
```

Comment:
get the soc Kconfigs and check them instead e.g. https://github.com/nrfconnect/sdk-nrf/blob/main/cmake/sysbuild/partition_manager.cmake#L497

Reply:
Done